src/lib/eina/meson.build

```diff
@@ -1,9 +1,4 @@
-if not sys_windows
-  eina_deps = [dl]
-else
-  eina_deps = [pcre_dep]
-endif
-
```

Comment:
This seems like a bad conflict resolution. Is it intentional?

Reply:
Yes. What we should do is change `dl` to be an empty declare_dependency. Meanwhile, `pcre_dep` is already a dependency of evil, which itself is a dependency of eina, which is a dependency of everyone directly or indirectly.

Reply:
you mean something like `dl = dependency()` at `/meson.build`? If so, shouldn't this PR handle it? Or at least let it how it is already is at `native_windows` and open an issue about this. I mean, this change will break the build won't it? As windows will try to use `dl` but won't find it, or I've missed something?

Reply:
Well, it builds here. And it is already handled in `header_checks/meson.build`: https://github.com/expertisesolutions/efl/blob/devs/expertise/native-windows/header_checks/meson.build#L163

Reply:
Also, it builds on CI too now *yay*

Reply:
oh nice!